### PR TITLE
Hotfix/new design/next bugs pack

### PR DESCRIPTION
**Pull request by Doctor-Cyclone**

### CODE REVIEW

File: src/app/features/history/components/history-view/history-view.component.ts [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class HistoryViewComponent {}
+export class HistoryViewComponent {
+  constructor(private readonly recentTradesStoreService: RecentTradesStoreService) {
+    this.readAllTrades();
+  }
+
+  public readAllTrades(): void {
+    setTimeout(() => this.recentTradesStoreService.updateUnreadTrades(true), 0);
```
Comment (axtezy) [inline src/app/features/history/components/history-view/history-view.component.ts:16]: зачем таймаут?

**Review by axtezy — CHANGES_REQUESTED** [review]

File: src/app/features/history/components/history-view/history-view.component.ts (hunk 1 above)
Comment (Doctor-Cyclone) [reply src/app/features/history/components/history-view/history-view.component.ts]: Done.

File: src/app/features/trade/components/settings-ccr/settings-ccr.component.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
         [max]="50"
         [ngModel]="formValue.slippageTolerance"
         [ngModelOptions]="{ standalone: true }"
-        [precision]="2"
```
[the remark this answers is not in the document]
Comment (Doctor-Cyclone) [reply src/app/features/trade/components/settings-ccr/settings-ccr.component.html]: Done.

File: src/app/features/trade/services/cross-chain/cross-chain.service.ts [hunk 3]
```diff
@@ -299,9 +297,7 @@ export class CrossChainService {
         ...(directTransaction && { directTransaction })
       };
 
-      try {
-        this.recentTradesStoreService.saveTrade(fromAddress, tradeData);
-      } catch {}
+      console.log(fromAddress, tradeData);
```
[the remark this answers is not in the document]
Comment (Doctor-Cyclone) [reply src/app/features/trade/services/cross-chain/cross-chain.service.ts]: Done.

**Review by axtezy — APPROVED** [review]

**Pull request merged**